### ISSUE TITLE
Anyone else getting this message? "Image was not able to get Liked! maybe blocked ?"

### ISSUE THREAD

**Issue by crowg** [InstaPy/InstaPy#4359]
--> Image was not able to get Liked! maybe blocked ?
--> Invalid Like Element!
I am getting this message on every pic that should be liked... Before anyone ask, I am not blocked on instagram. And I am checking to see if anyone else is getting the same thing and would anyone know why this would be coming up on every photo?

**Comment by RyukiMizo** [comment]
me too.. I got same message. 
my environment is ec2 localhost and I set proxy.

**Comment by scottgrayson** [comment]
I get it as well. "Invalid Like Element" makes it seem like Instagram updated their website and the "Like Element" selectors need to be updated for the script
https://github.com/timgrossmann/InstaPy/blob/master/instapy/like_util.py#L675

**Comment by ghost** [comment]
Same here:
--> Image was not able to get Liked! maybe blocked ?

**Comment by matteblacktrees** [comment]
Same for us. No liking capability.

**Comment by andrijasinski** [comment]
Same here in headless browser mode. Also, unfollowing does not work in a headless browser. 
> INFO [2019-04-27 13:24:00] [iamkatrln]  --> Already unfollowed 'elescosmetics'! or a private user that rejected your req

But the user is followed.

**Comment by crowg** [comment]
@timgrossmann

**Comment by Harosteadi** [comment]
Same here since 25th. Only in headless browser mode. I can like trough app and web so I think I am not banned. I also have the problem @andrijasinski mentioned for unfollow feature.

**Comment by Harosteadi** [comment]
#3676

**Comment by ghost** [comment]
Workaround: switch off headless browser mode (headless_browser=False)
Please update if it is possible to use the like feature again with headless browser mode on.

**Comment by andrijasinski** [comment]
@timgrossmann need your attention here. This bug blocks usage if non headless browser is not an option.

**Comment by zachcolbert** [comment]
Downgrade chrome browser to 72 and block auto update

**Comment by Harosteadi** [comment]
Many people found bugs with version 74 on MacOS. [An example](https://bugs.chromium.org/p/chromedriver/issues/detail?id=2870) _(open with chrome browser)_.


[Whole list of bugs posted](https://bit.ly/2Y5PeLh).

What I found and worked for me is **downgrade from v.74 to v.72** following these steps Ian Zabel posted on [GitHub](https://github.com/teamcapybara/capybara/issues/2181#issuecomment-487687514).

- Remove Chrome Browser (I personally used CleanMyMac)

- Download v.72 [here](https://google-chrome.en.uptodown.com/mac/old) (cannot find a version of Chrome 73 anywhere...)

- Run the following command to prevent chrome from having permissions to execute the updater script

     `sudo chmod -R 000 ~/Library/Google`

- (and it can’t update itself any more unless you allow it again with the command)

     `sudo chmod -R 755 ~/Library/Google`


I suppose that Google ChromeDriver will be updated soon with these bug fixed, for now, downgrade is the temporary solution.

**Comment by andrijasinski** [comment]
Thanks, will give it a try later

**Comment by SirCipher** [comment]
An alternative is to use Firefox and Gecko. This is working fine for me!

**Comment by crowg** [comment]
I updated Instapy 2 days ago from the 4 day ago update. Now the chrome driver does not work at all. i keep getting this message. 

"Message: unknown error: unable to discover open pages
  (Driver info: chromedriver=2.46.628411 (3324f4c8be9ff2f70a05a30ebc72ffb013e1a71e),platform=Mac OS X 10.13.6 x86_64)
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/instapy/browser.py", line 146, in set_selenium_local_session
    chrome_options=chrome_options)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/chrome/webdriver.py", line 81, in __init__
    desired_capabilities=desired_capabilities)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 157, in __init__
    self.start_session(capabilities, browser_profile)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 252, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: unable to discover open pages
  (Driver info: chromedriver=2.46.628411 (3324f4c8be9ff2f70a05a30ebc72ffb013e1a71e),platform=Mac OS X 10.13.6 x86_64)

Traceback (most recent call last):
  File "quickstart.py", line 16, in <module>
    headless_browser=False,)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/instapy/instapy.py", line 295, in __init__
    self.set_selenium_local_session()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/instapy/instapy.py", line 349, in set_selenium_local_session
    raise InstaPyError(err_msg)
instapy.exceptions.InstaPyError: ensure chromedriver is installed at /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/instapy_chromedriver/chromedriver_mac64"

**Comment by quintendewilde** [comment]
I have this on Rasberry pi? I'm not sure if there is a solution for this or not?? 

@wontfix ??
